packages/common/src/utils/validationUtils.ts

```diff
@@ -59,14 +59,23 @@ const getResult = (validation: CertificateDataValidation, data: CertificateData,
     question = data[questionId]
   }
 
+  if (validation.questions != null) {
+    if (validation.expressionType === 'OR') {
+      return validation.questions.some((v) => getResult(v, data, questionId))
+    }
+    return validation.questions.every((v) => getResult(v, data, questionId))
+  }
+
```

Comment:
Maybe refactor this like: 
 if (validation.questions != null) {
    return validation.expressionType === 'OR'
      ? validation.questions.some((v) => getResult(v, data, questionId))
      : validation.questions.every((v) => getResult(v, data, questionId))
  }